HowTos/create_vpc.rst

```diff
@@ -1,39 +1,58 @@
 .. meta::
    :description: VPC Network CIDR Management Tool
-   :keywords: Aviatrix VPC Tracker, AWS VPC
+   :keywords: Aviatrix VPC Tracker, AWS VPC, AZURE VNet
 
 ###################################
 Create a VPC
 ###################################
 
-Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ that consists of a public subnet and a private subnet in two AZs and an IGW in the region and account of your choice. 
+Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ in AWS or a `VNet <https://a.aviatrix.com/learning/glossary/vnet.php>`_ in AZURE in the region and account of your choice. In addition, starting from 6.1, this tool creates multiple route tables associated with public and private subnets. One use case is to allow traffic load balancing when Aviatrix Spoke gateways are deployed. 
 
-The VPC CIDR range is from /16 to /24. The VPC name cannot contain underscore character ("_"). 
+To configure, go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
 
-Go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
+The VPC/VNet CIDR range is from /16 to /24.
```

Comment:
These  are the only acceptable CIDR ranges /16 to /24  in Create a VPC. It applies whether Transit or Firenet options are selected.